src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1808,44 +1861,44 @@ void UsgsAstroLsSensorModel::losToEcf(
    double sampleUSGSFull = sampleCSMFull;
    double fractionalLine = line - floor(line);
 
-   // Compute distorted image coordinates in mm (sample, line on image (pixels) -> focal plane 
-   double isisNatFocalPlaneX, isisNatFocalPlaneY; 
+   // Compute distorted image coordinates in mm (sample, line on image (pixels) -> focal plane
+   double isisNatFocalPlaneX, isisNatFocalPlaneY;
    computeDistortedFocalPlaneCoordinates(fractionalLine, sampleUSGSFull, isisNatFocalPlaneX, isisNatFocalPlaneY);
 
    // Remove lens distortion
-   double isisFocalPlaneX, isisFocalPlaneY; 
+   double isisFocalPlaneX, isisFocalPlaneY;

```

Comment:
These variables should probably be updated...?

Reply:
Agreed. Good catch, I missed these.